### ISSUE TITLE
Link to github repo not visible

### ISSUE THREAD

**Issue by michalkalinowski-** [ariatemplates/wwwpublic#19]
Hello,

Can you enhance the front page so that the link to github repo is easily visible as it is the case in all other projects

for example:
https://angularjs.org

thanks :)

**Comment by benouat** [comment]
yes indeed you're right that we are missing the link from the homepage.
I'll work on fixing that !
